integration/form-test.ts

```diff
@@ -576,3 +576,64 @@ test.describe("Forms", () => {
     });
   });
 });
+
+test.describe("Forms in an app with custom routes", () => {
+  let fixture: Fixture;
+  let appFixture: AppFixture;
+
+  test.beforeAll(async () => {
+    fixture = await createFixture({
+      files: {
+        "remix.config.js": `
+          module.exports = {
+            ignoredRouteFiles: ["**/.*"],
```

Comment:
We're not defining any dotfiles, so we can remove this line and make the test as clean as possible

```suggestion
```